outlines/1.3-migration

```diff
@@ -0,0 +1,31 @@
+1. Breaking changes (as we'd normally do this)
+
+2. Recommended changes : Modules [link to app structure guide]
+  1. Applications: Move all files to imports, initialize in one spot, link via `import`
+  2. Import *all* symbols from packages, including meteor core packages.
```

Comment:
I think this is a really important question that we should answer before 1.3. I can imagine an 1.3 world where we have the same globally exported symbols (now marked deprecated, such as `Meteor.loginWithPassword`), and a new set of importable symbols that in some cases are a lot saner. 

Where should we bring this up?